toolkit/scripts/imggen.mk

```diff
@@ -93,6 +93,10 @@ ifeq ($(DISABLE_UPSTREAM_REPOS),y)
 imagepkgfetcher_extra_flags += --disable-upstream-repos
 endif
 
+ifeq ($(DISABLE_MARINER_REPOS),y)
+imagepkgfetcher_extra_flags += --disable-mariner-repos
```

Comment:
Please change the flag name to `--disable-default-repos`

Reply:
Good call, more generic.:)